mismi-s3/src/Mismi/S3/Data.hs

```diff
@@ -139,6 +144,18 @@ data Address = Address {
   , key :: Key
   } deriving (Eq, Ord)
 
+bucketL :: Lens' Address Bucket
+bucketL = lens bucket (\a b -> a { bucket = b })
+
+keyL :: Lens' Address Key
+keyL = lens key (\a k -> a { key = k })
+
+bucketTextL :: Lens' Bucket Text
+bucketTextL = lens unBucket (\b t -> b { unBucket = t })
+
+keyTextL :: Lens' Key Text
+keyTextL = lens unKey (\k t -> k { unKey = t })
+
```

Comment:
If we are going to export lenses, I think we should put them in a separate module and not automatically re-export them in global `Mismi.S3`. We have had a lot of name pollution problems with mismi lately, and I think we should be a little bit more conservative in what we export.